data-pipeline/examples/send-traces-with-stats.rs

```diff
@@ -55,6 +55,6 @@ fn main() {
         traces.push(trace);
     }
     let data = rmp_serde::to_vec_named(&traces).unwrap();
-    exporter.send(&data, 100).unwrap();
+    exporter.send(tinybytes::Bytes::from(data), 100).unwrap();
```

Comment:
Myabe I'm under the wrong assumption but wouldn't it be better to hide the tinybytes handling from the user so the signature of the public function would be like this:
```
pub fn send(&self, data: &[u8], trace_count: usize) -> Result<String, String>
``` 
This way we will handle the tinybytes wrapper inside the function and also avoid the dependency in the ffi layer.